actions/loadconfig.yaml

```diff
@@ -27,11 +27,18 @@ parameters:
         position: 3
     config_file:
         type: "string"
-        description: "configuration to load to device"
+        description: "Configuration file including full path to load to device (/x/y/z.type)"
         required: true
         position: 4
     method:
         type: "string"
         description: "Method to load the config (either 'merge' or 'replace', default is 'merge')"
         required: false
         position: 5
+    inline_transfer:
```

Comment:
I'd prefer this is `boolean` type. What is the expected type of the `device.inline_transfer` property?

Reply:
Have to admit to being lazy here. I tried boolean and couldn't get it to work at the time. Went for speed over correctness. I will punish myself appropriately.